app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
     include SessionsHelper
+
+    def current_user
+        User.find_by(id: session[:user_id])
+    end
```

Comment:
@hashimoto97 Delete this method, because we already have this one inside SessionsHelper